### PR TITLE
feat: symlink core aspects from a symlink that points to the current Bit CLI

### PR DESCRIPTION
**Pull request by zkochan**
Related PR in BVM: https://github.com/teambit/bvm/pull/88
Related fix in pnpm: https://github.com/pnpm/pnpm/pull/6854
new component created: https://bit.cloud/teambit/bvm/path

### CODE REVIEW

File: scopes/harmony/aspect-loader/core-aspects.ts [hunk 1]
```diff
@@ -22,10 +22,18 @@ function getAspectDirFromPath(id: string, pathsToResolveAspects?: string[]): str
...
   let dirPath;
 
+  if (currentBitDir) {
+    dirPath = resolve(currentBitDir, 'node_modules/@teambit', aspectName);
+    if (!existsSync(dirPath)) {
+      throw new Error(`unable to find ${aspectName} in ${dirPath}`);
+    }
+    return dirPath;
+  }
+
```
Comment (GiladShoham) [inline scopes/harmony/aspect-loader/core-aspects.ts:36]: @zkochan why not just pass the `currentBitDir` as the second arg to the `getAspectDirFromPath` below

File: scopes/dependencies/dependency-resolver/dependency-linker.ts [hunk 2]
```diff
@@ -580,6 +585,9 @@ export class DependencyLinker {
 
     const packageName = `@teambit/${name}`;
     const target = path.join(rootDir, 'node_modules', packageName);
+    if (this._currentBitDir) {
```
Comment (GiladShoham) [inline scopes/dependencies/dependency-resolver/dependency-linker.ts:588]: I wonder if we need to handle this also in some places of `load-bit.ts`
for example we have there `getMainAspect` which later used to link the `@teambit/bit` itself. which I think will now point to the wrong location.
you can also look on `linkBit` in this file, and see if you get the correct location or not in action.

Comment (zkochan) [reply scopes/dependencies/dependency-resolver/dependency-linker.ts:588]: yes, you are right. May I just update the getAspectDir to alway prefer reading from currentBitDir? Is there any scenario where currentBitDir should not be used?

Comment (GiladShoham) [reply scopes/dependencies/dependency-resolver/dependency-linker.ts:588]: @zkochan yes you can change the `getAspectDir`

**Pull request merged**